build/Tests/UnitTests/Foundation.WindowsOnly/Foundation.WindowsOnly.UnitTests.vcxproj

```diff
@@ -234,8 +234,6 @@
   </ItemGroup>
   <ItemGroup>
     <ClangCompile Include="$(StarboardBasePath)\Frameworks\Foundation\NSObject_NSCoding.mm" />
```

Comment:
It did, but it merged before Foundation.WindowsOnly

---
In reply to: [73439035](https://github.com/Microsoft/WinObjC/pull/806#discussion_r73439035) [](ancestors = 73439035,73436897,73436698)